lib/ansible/modules/network/iosxr/iosxr_interface.py

```diff
@@ -17,34 +17,47 @@
 ---
 module: iosxr_interface
 version_added: "2.4"
-author: "Ganesh Nalawade (@ganeshrn)"
+author:
+    - "Ganesh Nalawade (@ganeshrn)"
+    - "Kedar Kekan (@kedarX)"
 short_description: Manage Interface on Cisco IOS XR network devices
 description:
   - This module provides declarative management of Interfaces
     on Cisco IOS XR network devices.
 extends_documentation_fragment: iosxr
 notes:
-  - Tested against IOS XR 6.1.2
+  - Tested against IOS XRv 6.1.2
+  - Preconfiguration of physical interfaces is not supported with C(netconf) transport.
 options:
   name:
     description:
-      - Name of the Interface.
+      - Name of the interface to configure in C(type + path) format. e.g. C(GigabitEthernet0/0/0/0)
     required: true
   description:
     description:
-      - Description of Interface.
+      - Description of Interface being configured.
   enabled:
     description:
-      - Interface link status.
+      - Removes the shutdown configuration, which removes the forced administrative down on the interface,
+        enabling it to move to an up or down state.
+    choices: ['True', 'False']
+    default: True
+  active:
+    description:
+      - Whether the interface is C(active) or C(preconfigured)
```

Comment:
Would it be possible to go into some detail, rather than just repeating the configuration options

Reply:
yes